### PR TITLE
First draft of Add Event Form

### PR DESCRIPTION
**Pull request by InstincDev**
I would appreciate @Caleb-Cohen @intelagense and anyone else's thoughts on the form.
I will be changing the input for Recurring Dates and adding the Success pg tomorrow.

### CODE REVIEW

**Comment by Caleb-Cohen** [comment]
@InstincDev great work! Do we need to keep the .EJS files or can we remove them from this PR?

File: package.json [hunk 1]
```diff
@@ -32,5 +32,13 @@
     "passport-discord": "^0.1.4",
     "passport-local": "^1.0.0",
     "validator": "^13.6.0"
-  }
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/InstincDev/Together.git"
```
Comment (Caleb-Cohen) [inline package.json:38]: What is the functionality of this?

File: controllers/home.js [hunk 2]
```diff
@@ -2,4 +2,7 @@ module.exports = {
   getIndex: (req, res) => {
     res.render("index.ejs");
   },
+  getAdd: (req, res) => {
+    res.render("add_event.ejs");
```
Comment (Caleb-Cohen) [inline controllers/home.js:6]: Can we remove this?

**Review by Caleb-Cohen — COMMENTED** [review]
Added some comments to the lines

File: controllers/home.js (hunk 2 above)
Comment (InstincDev) [reply controllers/home.js:6]: Yes definitely.

File: package.json (hunk 1 above)
Comment (InstincDev) [reply package.json:38]: Not sure. Probably got added while I was trying to get the app running.  🙃  You can delete or I will in the morning.

**Comment by InstincDev** [comment]
Thanks. The EJS can be deleted or I will in the morning.  I forgot about them once I got everything working.

File: package.json (hunk 1 above)
Comment (Caleb-Cohen) [reply package.json]: removed it

**Pull request merged**